setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup, find_packages
+from CanDI.__version__ import version
+
+from pathlib import Path
+
+this_directory = Path(__file__).parent
+
+setup(
+    name='CanDI',
+    description='A cancer data integration package',
+    version=version,
+    packages=find_packages(),
+    python_requires='>=3.9',
+    install_requires=[
+        "pandas",
+        "configparser",
+        "requests",
+        "tqdm",
+    ],
+    url = 'https://github.com/GilbertLabUCSF/CanDI',
+    entry_points={
+        'console_scripts': [
+            'candi-install = CanDI.setup.install:main',
```

Comment:
`candi-install` works as a command.

```
usage: candi-install [-h] [--source SOURCE] [--data_dir DATA_DIR]

optional arguments:
  -h, --help           show this help message and exit
  --source SOURCE      Specify the download source
  --data_dir DATA_DIR  Specify the data directory
```

Reply:
@Yogiski – Hi Chris, hope all is well! I made some changes to CanDI. Let me know what you think!